src/main/java/io/electrum/moneytransfer/model/PaginationInfo.java

```diff
@@ -0,0 +1,163 @@
+package io.electrum.moneytransfer.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Pattern;
+import java.util.Objects;
+
+public class PaginationInfo {
+
+   @JsonProperty("nrRecords")
+   private Long nrRecords = null;
+
+   @JsonProperty("range")
+   private String range = null;
+
+   @JsonProperty("firstPage")
+   private String firstPage = null;
+
+   @JsonProperty("previousPage")
+   private String previousPage = null;
+
+   @JsonProperty("nextPage")
+   private String nextPage = null;
+
+   public PaginationInfo nrRecords(Long nrRecords) {
+      this.nrRecords = nrRecords;
+      return this;
+   }
+
+   /**
+    * The total number of records in this search
+    * minimum: 1
+    *
+    * @return nrRecords
+    **/
+   @JsonProperty("nrRecords")
+   @ApiModelProperty(value = "The total number of records in this search")
+   @Min(0)
```

Comment:
Here is a misalignment the comment says 1 and the code says 0. Which is it?

```suggestion
    * minimum: 1
    *
    * @return nrRecords
    **/
   @JsonProperty("nrRecords")
   @ApiModelProperty(value = "The total number of records in this search")
   @Min(0)
```

Reply:
Done. It's 0.